### ISSUE TITLE
Research: analysis

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#14518]

**Comment by JonathanKatora** [comment]
@r-bartlett-gsa  Do you think https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17920 Is a good candidate for this SE?

**Comment by r-bartlett-gsa** [comment]
> [@r-bartlett-gsa](https://github.com/r-bartlett-gsa) Do you think [#17920](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17920) Is a good candidate for this SE?

@JonathanKatora I don't think so. I would like to keep this epic focused on the map-specific enhancements. 17920 seems to be a larger facility locator enhancement, that we could plan for the upcoming quarters.

**Comment by r-bartlett-gsa** [comment]
@JonathanKatora This epic perhaps should be added to OCTO board as well.
